sandbox/src/mocks/patientServiceSearch/requests/ParametersReturningMultipleServices.json

```diff
@@ -0,0 +1,92 @@
+{
+  "resourceType": "Parameters",
+  "meta": {
+    "profile": [
+      "https://fhir.nhs.uk/STU3/StructureDefinition/eRS-PatientServiceSearch-Parameters-1"
+    ]
+  },
+  "parameter": [
+    {
+      "name": "requestType",
+      "valueCoding": {
+        "system": "https://fhir.nhs.uk/STU3/CodeSystem/eRS-RequestType-1",
+        "code": "APPOINTMENT_REQUEST"
+      }
+    },
+    {
+      "name": "sortBy",
+      "valueCoding": {
+        "system": "https://fhir.nhs.uk/STU3/CodeSystem/eRS-SortBy-1",
+        "code": "DISTANCE"
+      }
+    },
+    {
+      "name": "patient",
+      "valueIdentifier": {
+        "system": "http://fhir.nhs.net/Id/nhs-number",
+        "value": "9912003888"
+      }
+    },
+    {
+      "name": "referringClinician",
+      "valueIdentifier": {
+        "system": "http://fhir.nhs.net/Id/sds-user-id",
+        "value": "021600556514"
+      }
+    },
+    {
+      "name": "searchCriteria",
+      "resource": {
+        "resourceType": "Parameters",
+        "meta": {
+          "profile": [
+            "https://fhir.nhs.uk/STU3/StructureDefinition/eRS-ServiceSearchCriteria-Parameters-1"
+          ]
+        },
+        "id": "ServiceSearchCriteria-1",
+        "parameter": [
+          {
+            "name": "priority",
+            "valueCoding": {
+              "system": "https://fhir.nhs.uk/STU3/CodeSystem/eRS-Priority-1",
+              "code": "ROUTINE"
+            }
+          },
+          {
+            "name": "specialty",
+            "valueCoding": {
+              "system": "_baseUrl_/STU3/v1/CodeSystem/SPECIALTY",
+              "code": "CARDIOLOGY"
+            }
+          },
+          {
+            "name": "clinicType",
+            "valueCoding": {
+              "system": "https://fhir.nhs.uk/STU3/CodeSystem/eRS-ClinicType-1",
```

Comment:
Should this system be referencing the Code System API?

Reply:
Maybe we should bring it up as a potential change in the future